src/storage.cpp

```diff
@@ -1499,37 +1493,33 @@ namespace libtorrent
 	}
 #endif
 
-	int piece_manager::check_no_fastresume(storage_error& ec)
+	int piece_manager::check_no_fastresume(int flags, storage_error& ec)
 	{
-		bool has_files = false;
-		if (!m_storage->settings().get_bool(settings_pack::no_recheck_incomplete_resume))
```

Comment:
no, because the logic was moved to `piece_manager::check_fastresume` where it's a little more clear and convenient